### DISCUSSION TITLE
Describe tags & voabularies of .json in ontology?

### DISCUSSION THREAD

**Discussion by AngelaKranz** [nfdi4plants/nfdi4plants_ontology#16] [Ideas]
**_Question/Idea that was raised by @muehlhaus in the TA3 Meeting from 2021-08-10:_**

Will TA1 apply ontology to the describe the tags and vocabularies in the .json file, which is used as metadata for a metadata template (.xlsx)?
If yes, which ontology will be used and how it can be used? 

I would suggest that we include this into the discussion for the overall concept of the ontology

**Comment by xiaoranzhou** [comment]
> **_Question/Idea that was raised by @muehlhaus in the TA3 Meeting from 2021-08-10:_**
> 
> Will TA1 apply ontology to the describe the tags and vocabularies in the .json file, which is used as metadata for a metadata template (.xlsx)?
> If yes, which ontology will be used and how it can be used?
> 
> I would suggest that we include this into the discussion for the overall concept of the ontology

The current .json file will be depreciated. A new .json will be made by parsing the "metadate created from swate template". The new .json will inlcude ontology.
